homeassistant/components/minecraft_server/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "domain": "minecraft_server",
+  "name": "Minecraft Server",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/minecraft_server",
+  "requirements": [
+    "mcstatus==2.3.0"
+  ],
+  "ssdp": [],
+  "zeroconf": [],
+  "homekit": {},
+  "dependencies": [],
+  "codeowners": [
+    "@elmurato"
+  ]
```

Comment:
Remove `ssdp`, `zeroconf`, `homekit` keys + make one line for `codeowners` and `requirements`

Reply:
Done.